src/Compute/Compute.Sdk/Compute.Sdk.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PsModuleName>Compute</PsModuleName>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)..\..\Az.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Microsoft.Azure.PowerShell.Compute.Sdk</AssemblyName>
+    <RootNamespace>Microsoft.Azure.Management.Compute</RootNamespace>
+    <NoWarn>$(NoWarn);CS0108;CS1570;CS1572;CS1573</NoWarn>
```

Comment:
May I know if any reason you added CS1570 and CS1572?

Reply:
no i think i was just following what was in Dingmeng's changes in https://github.com/Azure/azure-powershell/tree/share/compute. let me remove that for now and see what happens